blogs/2022/03/28/increase-productivity-with-containers.md

```diff
@@ -0,0 +1,89 @@
+
+With all these reasons to be weary of the cloud, it can be uncomfortable to think about something as important as our development environment being hosted there.  So let’s stay comfortable! Let’s continue relying on our 6-year-old laptop that makes really weird whirring noises if you try to open an email too quickly, let alone try to build a project. And we’ll just comfortably wait for the inevitability of when the laptop crashes, and then once we get our new laptop, we have to rebuild our dev environment and try to remember exactly how we had it configured.
+
+![A meme showing a dog saying "This is fine" while sitting in a room on fire](this-is-fine-meme.png)
+
+Turns out staying in our comfort zone doesn’t sound so peachy either.
+
+While there are inherent risks with the cloud, it also comes with major benefits: scalability, faster performance, and easier maintenance, just to name a few. Hosting in the cloud gives you access to another machine’s resources that can then be spun up quickly and available anywhere. Combine that with the benefits of container-based development, and developers can be up and coding in the blink of an eye.
+
+## Containers in the cloud
+
+Running containers in the cloud is not a new concept; in fact, [one study](https://www.datadoghq.com/container-report/?utm_source=SocialMedia&utm_medium=Twitter&utm_campaign=OrganicPosting-containerreport) shows that at least half of all containers are running in the cloud. The basic infrastructure involves deploying a container to a cloud-hosted VM. For a development environment hosted in the cloud, the deployed container is the dev container that houses our code and dependencies. Developers can then connect to this VM to harness the power of containers in the cloud.
+
+VS Code’s entry into this space comes from powering [GitHub Codespaces](https://github.com/features/codespaces). Within minutes, you can create and configure a dev container hosted in the cloud that is ready whenever you need it. You can then connect via VS Code (in either the browser or desktop) to a development environment that is managed completely for you, no longer relying on your laptop’s resources to handle the demand.
+
+![A diagram showing the infrastructure and workflow of Codespaces](codespaces-diagram.png)
+
+Image Source: GitHub Codespaces overview - GitHub Docs
+
+## What’s next?
+
+As developers, we want to spend more time developing software and less time on headache-inducing configuration. Trends in the industry can, and should, be used to empower developers to increase their productivity. We’ve seen how containers and the cloud can take us to the next level – what do you think could be the next step in making our lives even easier?
```

Comment:
Do we want a CTA, i.e. check out Remote - Containers extension or doc for more?

Reply:
added CTA to the getting started documentation for remote containers